include/zmq.h

```diff
@@ -763,6 +766,8 @@ ZMQ_EXPORT int zmq_socket_monitor_versioned (
   void *s_, const char *addr_, uint64_t events_, int event_version_, int type_);
 ZMQ_EXPORT int zmq_socket_monitor_pipes_stats (void *s);
 
+ZMQ_EXPORT int zmq_ppoll (zmq_pollitem_t *items_, int nitems_, long timeout_, const sigset_t *sigmask_);
```

Comment:
It doesn't look like this (sigset_t) is compatible with windows, so I'd suggest to just ifdef both this and the signal.h include

Reply:
Ah right, makes sense, most builds were passing before, so this should fix most of them again!

Reply:
What if, to maintain full API in at least some way, we do this?

```suggestion
#if !defined _WIN32
ZMQ_EXPORT int zmq_ppoll (zmq_pollitem_t *items_, int nitems_, long timeout_, const sigset_t *sigmask_);
#else
ZMQ_EXPORT int zmq_ppoll (zmq_pollitem_t *items_, int nitems_, long timeout_, const void *sigmask_);
#endif
```

This way, it's still callable, but it will just return with errno ENOTSUP anyway on Windows, so the type doesn't really matter.

Reply:
I implemented this in the latest force push.